wolfssl/wolfcrypt/settings.h

```diff
@@ -849,12 +852,6 @@ extern void uITRON4_free(void *p) ;
 #endif /* FREESCALE_KSDK_MQX */
 
 #if defined(FREESCALE_FREE_RTOS) || defined(FREESCALE_KSDK_FREERTOS)
-    /* Allows use of DH with fixed points if uncommented and NO_DH is removed */
-    /* WOLFSSL_DH_CONST */
-    /* Allows use of DH with fixed points if uncommented and NO_DH is removed */
-    /* WOLFSSL_DH_CONST */
-    /* Allows use of DH with fixed points if uncommented and NO_DH is removed */
-    /* WOLFSSL_DH_CONST */
```

Comment:
The cleanup is not related to the DEOS port. But yes, I wanted to remove the three lines because they were unnecessary duplicate comments. I think one comment which is a few lines down should be sufficient. Please let me know if I missed something.